pybamm/citations.py

```diff
@@ -256,9 +256,7 @@ def print_citations(filename=None, output_format="text", verbose=False):
     """See :meth:`Citations.print`"""
     if citations._citation_err_msg is not None:
         raise ImportError(
-            f"Citations could not be registered. If you are on Google Colab - "
-            "pybtex does not work with Google Colab due to a known bug - "
-            "https://bitbucket.org/pybtex-devs/pybtex/issues/148/. "
+            f"Citations could not be registered."
```

Comment:
Could you please take a look at https://github.com/pybamm-team/PyBaMM/pull/2547 and just revert all the changes in the PR? That way we can get rid of some dead code too.

Reply:
Feel like I'm out of sync here. Why do we need to do this?

Reply:
This was because we had to check if `pybtex` was working on every `pybamm.print_citations()` or `citations.register()` call. Since it now works on Google Colab, we don't need to keep the try-except blocks anymore.